### PR TITLE
Fixes txt & pdf

### PR DESCRIPTION
**Pull request by tcosentino**
This uses the txt fix from the other PR and fixes pdf by using font-awesome pdf (rather than atom's default)

### CODE REVIEW

**Comment by tcosentino** [comment]
Oof didn't mean to change style.css like that, fixing..

File: css/style.css [hunk 1]
```diff
@@ -98,3115 +98,545 @@
 /* ----------------------------[ Fonts ]---------------------------------- */
 
 @font-face {
-  font-family: FontAwesome;
-  font-weight: normal;
-  font-style: normal;
-  src: url("../fonts/fontawesome.woff2");
+	font-family: FontAwesome;
+	font-weight: normal;
```
Comment (Alhadis) [inline css/style.css]: `un-prettyify`...? :p Heh, that's quite ironic. The tabs were already pretty, 2-spaces made it the opposite. 😁

**Comment by tcosentino** [comment]
😄 to each their own

**Comment by websemantics** [comment]
Thanks guys for your contribution, glad you find it useful :D

**Comment by tcosentino** [comment]
Yes, thank you for putting it together!

**Pull request merged**